src/mqtt/event_cam.rs

```diff
@@ -210,6 +212,16 @@ impl EventCamThread {
             camera: arc_cam.clone(),
         };
 
+        let mut snap_thread = SnapThread {
```

Comment:
BTW, on my camera (PoE Duo with Floodlight) this is causing a bunch of warnings I think:

```
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Remaining: 0 of 100 message space for 18 (ID: 109)
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Reaching limit of channel
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Remaining: 0 of 100 message space for 18 (ID: 109)
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Reaching limit of channel
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Remaining: 0 of 100 message space for 18 (ID: 109)
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Reaching limit of channel
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Remaining: 0 of 100 message space for 18 (ID: 109)
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Reaching limit of channel
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Remaining: 0 of 100 message space for 18 (ID: 109)
[2023-06-08T02:41:39Z WARN  neolink_core::bc_protocol::connection::bcconn] Reaching limit of channel
```

Reply:
Also I'm wondering if this thread should be conditional, only if the camera feature is enabled or we detect it via type on the device? Because my floodlight (w/o cameras) gets a whole bunch of these which keeps restarting the thread:

```
2023-06-08T02:42:45Z ERROR neolink::mqtt::event_cam] Snap thread aborted: Camera responded with Service Unavaliable
[2023-06-08T02:42:45Z WARN  neolink::mqtt::event_cam] Camera thread error: Camera responded with Service Unavaliable. Restarting...
```

Reply:
Is this on my last push. Because I added something to make it not restart when we get the not supported reply?

Reply:
Ah yours is service unavailable not unintelligible reply. Ok pushed.

Reply:
What's I'm thinking to do since detection is a bit flakey. Is to just send the message requesting the picture/floodlight and if the camera replies can't do that then give up as unsupported.